base/reflection.jl

```diff
@@ -73,25 +73,28 @@ function fullname(m::Module)
 end
 
 """
-    names(x::Module; all::Bool = false, imported::Bool = false)
+    names(x::Module; qualified::Bool=true, all::Bool = false, imported::Bool = false)
```

Comment:
Did we decide to try making new keywords here correspond somehow to the flags in `jl_binding_t` so that the filtering behavior of `names()` is much easier to specify?  I forgot which conclusion we came to!